### PR TITLE
[new release] encore (0.5)

### PR DESCRIPTION
**Pull request by dinosaure**
Library to generate encoder/decoder which ensure isomorphism

- Project page: <a href="https://github.com/mirage/encore">https://github.com/mirage/encore</a>
- Documentation: <a href="https://mirage.github.io/encore/">https://mirage.github.io/encore/</a>

##### CHANGES:

- Move to `bigarray-overlap` to be compatible with > MirageOS 3.6 and `js_of_ocaml` (mirage/encore#23)
- Update to `angstron.0.14.0` (mirage/encore#22)

### CODE REVIEW

File: packages/encore/encore.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune"
```
Comment (mseri) [inline packages/encore/encore.0.5/opam:25]: The dune-project file requires dune 2.0

**Comment by mseri** [comment]
I am not 100% sure what is going on with the CI...

File: packages/encore/encore.0.5/opam (hunk 1 above)
Comment (mseri) [reply packages/encore/encore.0.5/opam]: Can you upstream the change?

Comment (dinosaure) [reply packages/encore/encore.0.5/opam]: Done :+1:.

**Comment by kit-ty-kate** [comment]
All green. Thanks! Could you upstream the change? You said it's done but I don't see the commit on github.

**Pull request merged**